cisco_aci/datadog_checks/cisco_aci/fabric.py

```diff
@@ -128,7 +131,7 @@ def process_eth(self, node):
             eth_id = eth_attrs['id']
             tags = self.tagger.get_fabric_tags(e, 'l1PhysIf')
             tags.extend(common_tags)
-            if PY3:
+            if PY3 and self.enable_ndm:
```

Comment:
One thing I'm wondering, would it be possible to only check `self.enable_ndm` at the point where we'd submit the metadata?

It might be slightly less performant but the readability would be improved and there would be less checks to maintain

Reply:
i agree with you on the readability and was considering it 

it may not be super signficant in the end but i know we had some complaints from some customers already on the integration before we touched anything for metadata being sent

i can do that for now and try to work on that with the future ticket for improving performance

Reply:
or rather, i put the check in one helper fn so that it's easier to change if anything